src/utils/InputWrapper.js

```diff
@@ -76,7 +76,10 @@ InputWrapper.textProp = 'value';
  */
 InputWrapper.focus = function() {
     if (InputWrapper.hiddenInput) {
-        InputWrapper.hiddenInput.focus();
+        // the hidden input element only gets focused when the focus function is called within a timeout function
+        window.setTimeout(function () {
+            InputWrapper.hiddenInput.focus();
+        }, 0);
```

Comment:
It is not needed anymore. The problem was in InputControl.js. I fixed it in my latest commit.